modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -557,4 +557,104 @@ definition {
 		}
 	}
 
+	@description = "LPS-173894 -Verify that it's possible to send an e-mail notification with Action (Update)"
+	@priority = 5
+	test CanSendEmailViaActionUpdate {
+		property custom.properties = "feature.flag.LPS-173894=true";
+		property portal.acceptance = "true";
+		property test.name.skip.portal.instance = "ObjectNotifications#CanSendEmailViaActionUpdate";
+
+		task ("Given: an Email Notification Template") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "Email Body",
+				entryTo = "test@liferay.com",
+				fromAddress = "Liferay",
+				fromName = "Test Test",
+				name = "Notification Template",
+				notificationType = "Email",
+				subject = "Subject");
+
+			PortletEntry.save();
+		}
+
+		task ("And: an Custom Object with an Action (Update Entry)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 178782",
+				objectName = "CustomObject178782",
+				pluralLabelName = "Custom Objects 178782");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "true",
+				objectName = "CustomObject178782");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject178782");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 178782");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionLabel = "Custom Action",
+				actionName = "CustomAction",
+				active = "true",
+				notificationName = "Notification Template",
+				objectLabel = "Custom Object 178782",
+				thenAction = "Notification",
+				whenAction = "On After Update");
+
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customObjectField",
+				objectName = "CustomObject178782",
+				value = "Entry Test");
+		}
```

Comment:
Split this into two tasks, one to create the object and another to create the action. We want to be descriptive here so that we can analyze the test quickly by reading the task. Make sure you move the macro to add the entry with the task that creates the object definition.
And given a custom object with a text field and an object entry
And given an Action to send an email notification after updating an object entry